assets/swaggerhub_api.yaml

```diff
@@ -1176,15 +1194,17 @@ definitions:
       min_altitude:
         description: >-
           The UVR does not affect flights below this altitude
-        allOf:
-          - $ref: '#/definitions/Altitude'
+        # Note that this description does not render properly.  See the link
```

Comment:
Could you also change the type for message_id in UVR? To
      message_id:
        type: string
        format: uuid
        description: >-
          id.
        maxLength: 36
        minLength: 36

Reply:
In addition to message_id, the permitted_gufis field still references UUID. If you're changing message_id could make the same change similarly.

Reply:
Done and done.